### PR TITLE
JBIDE-24933 OC setting for connection ITest

### PR DESCRIPTION
**Pull request by jkopriva**
Signed-off-by: Josef Kopriva <jkopriva@redhat.com>

# Pull Request Checklist
## General
* Is this a blocking issue or new feature? If yes, QE needs to +1 this PR

## Code
* Are method-/class-/variable-names meaningful?
* Are methods concise, not too long?
* Are catch blocks catching precise Exceptions only (no catch all)?

## Testing
* Are there unit-tests?
* Are there integration tests (or at least a jira to tackle these)?
* Is the non-happy path working, too?
* Are other parts that use the same component still working fine?

## Function
* Does it work?

### CODE REVIEW

**Comment by odockal** [comment]
@jkopriva also, if you by chance have a green build where the test is executed (even some smoke suite would be sufficient) it would be perfect!

File: itests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/connection/v3/SetOCForNewConnectionTest.java [hunk 1]
```diff
@@ -0,0 +1,240 @@
...
+	private static final String OC_370_URL_WINDOWS = "https://github.com/openshift/origin/releases/download/v3.7.0/openshift-origin-client-tools-v3.7.0-7ed6862-windows.zip";
+	private static final String OC_370_URL_MAC = "https://github.com/openshift/origin/releases/download/v3.7.0/openshift-origin-client-tools-v3.7.0-7ed6862-mac.zip";
+	private static final String PROPERTY_OC_LOCATION = "OC client path";
+	private String pathToOC;
+	Path ocTestDirectory = Paths.get("target", "octest");
+
+	//@InjectRequirement
+	//private OpenShiftCommandLineToolsRequirement ocRequirement;
+
+	public String downloadOCBinary(boolean latestOC, String destinationDirectory) {
```
Comment (odockal) [inline itests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/connection/v3/SetOCForNewConnectionTest.java]: I was kinda aiming for a suggestion that if we wont use that requirement directly, we might want to extract useful methods from it into some utility class and keep ocbinary requirement simple and just calling methods from such class. wdyt?

File: itests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/connection/v3/SetOCForNewConnectionTest.java [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+/**
+ * Implements JBIDE-24933
+ * @author jkopriva@redhat.com
+ */
+@RunWith(RedDeerSuite.class)
+@OpenPerspective(JBossPerspective.class)
+//@OCBinary(cleanup = true)
```
Comment (odockal) [inline itests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/connection/v3/SetOCForNewConnectionTest.java]: Please, remove those commented code and the code below.

File: itests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/connection/v3/SetOCForNewConnectionTest.java (hunk 1 above)
Comment (jkopriva) [reply itests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/connection/v3/SetOCForNewConnectionTest.java]: @odockal we can move it, but I really do not see any use case for it (at least for now).

Comment (odockal) [reply itests/org.jboss.tools.openshift.ui.bot.test/src/org/jboss/tools/openshift/ui/bot/test/connection/v3/SetOCForNewConnectionTest.java]: Sure thing.. ;)

**Comment by jkopriva** [comment]
green build: https://dev-platform-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/openshift.itests/422/jdk=openjdk-1.8,label=rhel7/

**Comment by jkopriva** [comment]
@odockal thx for review :)

**Comment by odockal** [comment]
+1. Merging.

**Pull request merged**